File: preliminary_analysis/baseline_cpp_results/instruct_generations/cpp/files_before_feedback/question-1-pass-1.cpp
#include <bits/stdc++.h> 
 using namespace std ;
bool sumOfTwoCubes(int n) {
    int x = (int)sqrt(n);
    if (x * x * x == n) return true;
    for (int i = x + 1; i <= n; i++) {
        int y = n - i * i;
        if (y * y * y == i * i) return true;
    }
    return false;
}

File: fine_tuning/results/test_output_files/question-1-pass-1.py
import math


def sumOfTwoCubes(n):
    for i in range(1, int(math.sqrt(n))):
        j = int(math.pow(n - i, 1.0 / 3))
        if (i * i * i + j * j * j == n):
            print("Yes")
            return
    print("No")


if __name__ == "__main__":
    sumOfTwoCubes(12)
